src/channel-slider/README.md

```diff
@@ -126,8 +126,19 @@ All attributes are reactive:
 | `step` | `step` | `number` | Computed automatically based on `this.min` and `this.max`. | The granularity that the slider's current value must adhere to. |
 | `value` | `value` | `number` | `(this.min + this.max) / 2` | The current value of the slider. |
 | `color` | `color` | `Color` &#124; `string` | `oklch(50% 50% 180)` | The current color value. |
-| - | `minColor` | `Color` | `oklch(0% 50% 180)` | The minimum color value _(read-only)_. |
-| - | `maxColor` | `Color` | `oklch(100% 50% 180)` | The maximum color value _(read-only)_. |
+
+### Getters
+
+These properties are read-only.
+
+| Property | Type | Description |
+|----------|------|-------------|
+| `minColor` | `Color` | The color corresponding to the minimum value of the slider. |
+| `maxColor` | `Color` | The color corresponding to the maximum value of the slider. |
+| `stops` | `Array<Color>` | The array of the slider color stops with `minColor` and `maxColor` as its first and last elements, respectively. |
+| `progress` | `number` | The slider progress, as a number between `0` and `1`, calculated on its current value (e.g. `0.42`). |
```

Comment:
```suggestion
| `progress` | `number` | The slider value converted to a 0-1 number with `0` corresponding to the min of the range and `1` to the max. |
```

Reply:
We should probably update the corresponding description for `<color-slider>`, too.